internal/fourslash/_scripts/convertFourslash.mts

```diff
@@ -1439,6 +1461,71 @@ function parseBaselineSmartSelection(args: ts.NodeArray<ts.Expression>): Cmd {
     };
 }
 
+function parseSemanticClassificationsAre(args: readonly ts.Expression[]): [VerifySemanticClassificationsCmd] | SkipStatement | undefined {
+    if (args.length < 1) {
+        console.error("semanticClassificationsAre requires at least a format argument");
+        return undefined;
+    }
+
+    const formatArg = args[0];
+    if (!ts.isStringLiteralLike(formatArg)) {
+        console.error("semanticClassificationsAre first argument must be a string literal");
+        return undefined;
+    }
+
+    const format = formatArg.text;
+
+    // Only handle "2020" format for semantic tokens
+    if (format !== "2020") {
+        // Skip other formats like "original" - return sentinel to continue parsing

```

Comment:
I don't remember what the distinction was, but it feels like you should just mark these as "unparsed" for later